src/track/track.cpp

```diff
@@ -1103,6 +1118,30 @@ void Track::removeCuesOfType(mixxx::CueType type) {
     }
 }
 
+void Track::swapHotcues(int a, int b) {
+    VERIFY_OR_DEBUG_ASSERT(a != b) {
+        qWarning() << "Track::swapHotcues rejected," << a << "==" << b;
+        return;
+    }
+    VERIFY_OR_DEBUG_ASSERT(a > Cue::kNoHotCue || b > Cue::kNoHotCue) {
```

Comment:
why `>` instead of `!=`?

Reply:
Because **I know** that hotcue indices are valid only if they are > 
Of course that might change so != is safer.

Reply:
done.